tests/Fixer/Basic/NonPrintableCharacterFixerTest.php

```diff
@@ -22,33 +22,26 @@
 final class NonPrintableCharacterFixerTest extends AbstractFixerTestCase
 {
     /**
-     * @param string       $expected
-     * @param string       $input
-     * @param \SplFileInfo $file
+     * @param string      $expected
+     * @param null|string $input
      *
      * @dataProvider provideExamples
      */
-    public function testFix($expected, $input, $file)
+    public function testFix($expected, $input = null)
     {
-        $this->doTest($expected, $input, $file);
+        $this->doTest($expected, $input);
```

Comment:
FYI: even when the content was read from file, it's not obligatory to pass the file to `doTest`.